pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/model/elastic/migration/BaseCollectionMigratorTest.java

```diff
@@ -30,18 +30,18 @@ public void transformObject() {
     private List mockOncHistoryDetail() {
         OncHistoryDetail oncHistoryDetail =
                 new OncHistoryDetail(23, 0, null, null, null, null, null, null, null, null, null, null, null, null, null,
-                        null, null, null, null, null, null, null, null, null, mockTissues(), null, null, false, null, null, 0);
+                        null, null, null, null, null, null, null, null, null, mockTissues(), null, null, false, null, null, 0, null);
         return Collections.singletonList(oncHistoryDetail);
     }
 
     private List<Tissue> mockTissues() {
         List<Tissue> fieldValue = new ArrayList<>(List.of(new Tissue(11, 22,
```

Comment:
These constructors are really hard to use.  Can you create new constructors that only use the non-null constructors?  That will save many lines of code and make it easier for us to understand which fields are actually required in combination with (or absence of) other fields.